static/js/debrief.js

```diff
@@ -1,26 +1,8 @@
 var nodesOrderedByTime;
 var visualizeInterval;
+var downloadType;
```

Comment:
you could probably avoid needing to make this a global scope var by following this pattern in the report file writing callback https://www.jstips.co/en/javascript/passing-arguments-to-callback-functions/ 

Up to you which approach you think is cleaner (as is, or with the callback closure).

Reply:
Callback closure feels pretty clean :)